### DISCUSSION TITLE
Facing issues while setting up Quml-Player as a web component in a React app

### DISCUSSION THREAD

**Discussion by Xitija** [Sunbird-inQuiry/Community#35] [Q&A] [answered]
Hello All,

I am trying to use the Quml-Player as a Web component in a sample React app created with
`npx create-react-app my-app`
I have imported the `sunbird-quml-player.js` file in the app.
I am facing the following errors - 
![qumlply](https://user-images.githubusercontent.com/65657373/191983272-1f9aa996-9bfa-49df-8a71-3f6c17a67712.png)

Can anyone please guide me here?
cc: @baljeetsingh @itsvick

**Comment by pallakartheekreddy** [comment, marked as answer]
@Xitija This is a valid bug. We have created a bug for the same. The team is working on this issue. Once it is fixed we will update the same thread.

https://project-sunbird.atlassian.net/browse/IQ-155
